core/write.go

```diff
@@ -10,7 +10,24 @@ import (
 // the tag used to denote the name of the question
 const tagName = "survey"
 
+// add a few interfaces so users can configure how the prompt values are set
+type settable interface {
+	Set(value interface{}) error
```

Comment:
I wonder if this should be something like `SetAnswer` instead.  The kingpin library has a similar interface `Set(string)` which would potentially conflict with this so a type could not be both compatible with kingpin and survey in this case.